### ISSUE TITLE
[schelp CSS] a.footnote adds the footnote number in the next line

### ISSUE THREAD

**Issue by prko** [supercollider/supercollider#3880]
a.footnote in scd.css adds the footnot number in the next line
<img width="572" alt="screenshot 2018-07-14 23 00 45" src="https://user-images.githubusercontent.com/416281/42725191-986afd8c-87ba-11e8-9a9f-c60ca357d9fc.png">

Until 3.9.3, it is displayed correctly:
<img width="584" alt="screenshot 2018-07-14 23 05 37" src="https://user-images.githubusercontent.com/416281/42725195-b2ef8c5e-87ba-11e8-8ac3-3052ac627ed7.png">

What I don't understand is the definition of a.footnote in the SC3.9.3 and the definition of a.footnote in the commit-hash 233ebeb are the same.
CSS
a.footnote {
    font-size: 9pt;
    top: -0.1em;
}
```
After finding this weirdness, I canceld my PR for #3864.
I will reopen later after proofing other things.

**Comment by prko** [comment]
This is observed on macOS. 
Under Linux and Windows, I have not tested the commit 233ebeb4b5dfe90f0a5d3afb9f50e2ac16a3a277

**Comment by prko** [comment]
The same occurs under windows. tested with commit-hash 8dbeb67

**Comment by khoin** [comment]
This is observed under Windows (10, at least).

**Comment by khoin** [comment]
The problem lies in this CSS declaration, shown in the screenshot. Seems to be from here: https://github.com/supercollider/supercollider/blob/develop/HelpSource/scdoc.css#L355

I won't touch it since I don't know if other formattings depend on it.
![image](https://user-images.githubusercontent.com/4620399/51966921-c854cf00-2422-11e9-940c-640fc6d5ac97.png)

**Comment by khoin** [comment]
@snappizz Can you take a look at this? 

I believe https://github.com/supercollider/supercollider/commit/b0e5ebfbc8e9d5ebb28dd28443115dbde6d6c021 caused this. And that commit was to solve this: https://github.com/supercollider/supercollider/issues/3858

**Comment by snappizz** [comment]
hi, thanks for being inquisitive, unfortunately i can't attend to this right now. if you have a solution please feel free to file a PR.

**Comment by brianlheim** [comment]
Closed in #4365, thanks @khoin !
